src/test/resources/pages/instructorFeedbackResultsPageOpen.html

```diff
@@ -4912,4 +4920,4 @@
 </div>
 
 
-<div class="tooltip fade top in" style="top: {*}px; left: {*}px; display: block;"><div class="tooltip-arrow"></div><div class="tooltip-inner">View results by sections</div></div></body></html>
```

Comment:
Godmode. Whenever a tooltip appears when hovered over, this element is updated. It's ignored in our html tests.